src/main/java/seedu/address/model/ResumeBook.java

```diff
@@ -139,7 +148,8 @@ public void setResumeToDisplay() {
      * Replaces the user profile detail with that of {@code person}.
      */
     public void setUser(Person user) {
```

Comment:
Pretty confusing here actually, since for the item lists like `internships` or `projects` the corresponding setters are `setInternships` and `setProjects`.

But then I realized these lists wrap around an internal observable list, while `observableUser` is not. So I went and changed the name.